lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.spec.ts

```diff
@@ -557,6 +557,13 @@ describe('CardViewTextItemComponent', () => {
             expect(component.property.value).toEqual(component.editedValue);
         });
 
+        it('should perform undo action by clearing the text that we enter in the text field using undo keyboard shortcut', async () => {
```

Comment:
you should test also other cases for example negative case where setValue is not called

Reply:
added 3 more test cases where control key is not pressed and keycode is not correct. also added test case for MacOS